data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/consumer/KafkaSourceCustomConsumer.java

```diff
@@ -161,10 +162,16 @@ public <T> void consumeRecords() throws Exception {
             topicMetrics.getNumberOfPollAuthErrors().increment();
             Thread.sleep(10000);
         } catch (RecordDeserializationException e) {
-            LOG.warn("Deserialization error - topic {} partition {} offset {}, seeking past the error record",
-                     e.topicPartition().topic(), e.topicPartition().partition(), e.offset(), e);
+            LOG.warn("Deserialization error - topic {} partition {} offset {}",
+                     e.topicPartition().topic(), e.topicPartition().partition(), e.offset());
+            if (e.getCause() instanceof AWSSchemaRegistryException) {
+                LOG.warn("AWSSchemaRegistryException. Retrying after 30 seconds");
```

Comment:
Can we also print exception message, it'll help customer to fix the issue e.g. if its access denied.